.github/workflows/test.yml

```
@@ -23,8 +23,8 @@ jobs:
       - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
     
-      - name: Execute Gradle test
-        run: ./gradlew test check
+      - name: Execute Gradle build
+        run: ./gradlew build
```

nit: To see build errors just in case.
```suggestion
        run: ./gradlew build --stacktrace
```